raster/r.surf.gauss/testsuite/test_surf_gauss.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env python3
+
+"""
+MODULE:    Test of r.surf.gauss
+
+AUTHOR(S): Corey White <ctwhite48 gmail com>
+
+PURPOSE: Tests random gauss surface generation
+
+COPYRIGHT: (C) 2023 by Corey White and the GRASS Development Team
```

Comment:
```suggestion
COPYRIGHT: (C) 2024 by Corey White and the GRASS Development Team
```

Wasn't merged in 2023. Or if it was written in 2023, add a date range

Reply:
I set the date range to 2023  - 2024.